subsys/net/lib/sockets/sockets.c

```diff
@@ -17,6 +14,11 @@ LOG_MODULE_REGISTER(net_sock, CONFIG_NET_SOCKETS_LOG_LEVEL);
 #include <zephyr/net/net_pkt.h>
 #include <zephyr/net/socket.h>
 #include <zephyr/net/socket_types.h>
+#ifdef CONFIG_ARCH_POSIX
```

Comment:
Does CONFIG_ARCH_POSIX make the zephyr/posix names stop working?

Reply:
No

Reply:
oh, so for CONFIG_ARCH_POSIX, you *need* to use the non-zephyr names to ensure you reach the OS versions?